docs/process/testing.md

```diff
@@ -86,24 +86,25 @@
   - [ ] Publish Repository modal present if repo is unpublished and user clicks `Publish repository` button
     - [ ] GitHub tab is default; Enterprise tab is also present
       - [ ] User must be signed in to publish, else `Sign In` button present on tab
-    - [ ] Modal fields are Name, Description (optional), `Keep this code private` checkbox, Organization list (alpha order, tab-specific), `Cancel` button, `Publish Repository` button
+    - [ ] Modal fields are Name, Description (optional), `Keep this code private` checkbox, Organization list (alpha order, tab-specific, if orgs exist), `Cancel` button, `Publish Repository` button
       - [ ] Clicking `Publish Repository` button pushes repo to GitHub.com or Enterprise; `Cancel` button closes modal
         - [ ] Repository is present on GitHub.com or Enterprise if published 
 	- [ ] 'Visibility can't be private error' shown, if user's plan is not sufficient
 	- [ ] A `.gitattributes` file is added to the repository as part of the initial commit
+	- [ ] If repo is forked and upstream remote does not match, then modal is surfaced with Ignore/Update button 
 
 ### Application
   - [ ] Minimize, Maximize, Close buttons work in top nav
       - [ ] If user zooms in and quits app, settings should be retained when reopened
   - [ ] Double-clicking local desktop icon opens the application (Mac only)
-  - [ ] Double clicking top nav bar toggles full-screen / last used screen-size (Mac only); Exit by (Mac: ^⌘f11; Windows: Alt)
+  - [ ] Double clicking top nav bar toggles full-screen / last used screen-size (Mac only); Exit by (Mac: `^⌘f11`; Windows: `Alt`)
```

Comment:
Should this be `F11`?

Reply:
^⌘f11 for mac, believe Alt for Windows

Reply:
I prefer using `<kbd>` for key combinations. You may want to consider it for your documentation? It's kind of a cool trick: <kbd>^⌘f11</kbd>

Reply:
Holy 🐮, didn't know about that one.

Reply:
Nice, for next time! Thanks